performance/images.md

```diff
@@ -82,7 +81,7 @@ For content images, we encourage using an on-premise or cloud-based image server
 * Ensure that all images are optimised automatically.
 * Ensure that the appropriate cache headers are always set.
 
-Additionally, future improvements to these tools will result in improvements to the images being served automatically, without the images having to be manually re-encoded.
+As an additional benefit, future improvements to these tools may result in improvements to the images being served automatically, without the images having to be manually re-encoded.
```

Comment:
```suggestion
As an additional benefit, changes to these tools may result in improvements to the images being served automatically, without the images having to be manually re-encoded.
```
Just tweaking it to remove some redundancy and improve the way it reads.

Separately, is there any link that can explain why this statement is made?